native_client/ctcdecode/ctc_beam_search_decoder.h

```diff
@@ -135,6 +144,7 @@ ctc_beam_search_decoder_batch(
     double cutoff_prob,
     size_t cutoff_top_n,
     std::shared_ptr<Scorer> ext_scorer,
+    std::set<std::string, float> hot_words,
```

Comment:
you use `std::set` here it's not consisent with `std::map` of the non batch call

Reply:
fixed in 125a59abbe068a3bc2bbfbc3c443318590bc62a4